apps/website/src/content/docs/typography.mdx

```diff
@@ -1,17 +1,56 @@
 ---
 title: Typography
-description:
+description: Typography components provide consistency in text styling across the application.
 thumbnail: Typography
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='text--basic' />
-
 ## Text
 
-<LiveExample src='Text.main.jsx' truncate={false}>
-  <AllExamples.TextMainExample client:load />
+`Text` is a [polymorphic](#polymorphic-only-semantics-and-not-size) typography component that can be used to render any kind of text as any kind of element.
+
+```jsx
+<Text>This is a text.</Text>
+```
+
```

Comment:
I think we should have a paragraph here that describes the `variant` and `as` props at a higher-level, and mentions the defaults.

Reply:
I have added a paragraph talking about these two props.